java/src/main/java/api/example/java/controllers/BaseController.java

```diff
@@ -0,0 +1,54 @@
+package api.example.java.controllers;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpSession;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.reactive.function.client.WebClientResponseException;
+
+import api.example.java.Config;
+import api.example.java.model.TokenResponse;
+
+public class BaseController {
+
+	@Autowired
+	protected Config config;
+
+	public BaseController() {
+	}
+
+	protected String getAccessTokenFromSession(HttpServletRequest request) {
+		TokenResponse tokenResponse = (TokenResponse) request.getSession().getAttribute("tokenResponse");
+		return tokenResponse.getAccessToken();
+	}
+
+	protected boolean isUserLoggedIn(HttpSession session) {
+		Object tokenResponse = session.getAttribute("tokenResponse");
+		if (tokenResponse != null) {
+			return true;
+		}
+	
+		return false;
```

Comment:
You code simplify line 29-33 to return tokenResponse != null;

Reply:
fixed